### ISSUE TITLE
oscillating reward with the default qwen3_1_7b.yaml recipe

### ISSUE THREAD

**Issue by sfc-gh-kganesan** [meta-pytorch/forge#560]
### 🐛 Describe the bug

Hi Team,
Great work with torchforge!  
I am trying to get a decent reward graph for gs8k recipe and running into a reward oscillation issue.
<img width="834" height="320" alt="Image" src="https://github.com/user-attachments/assets/4bf6ce0f-0286-4611-bab5-6051b228f475" />

```python -m apps.grpo.main --config apps/grpo/qwen3_1_7b.yaml```
using the default recipe the reward is oscillating it would be great if the hyperparameters used or any other setup related info that needs to be taken care can be provided

**Comment by felipemello1** [comment]
closing it in favor of https://github.com/meta-pytorch/torchforge/issues/556
